main.py

```diff
@@ -1,10 +1,59 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for user, info in data.items():
+        nickname = user
+        email = info["email"] if info["email"] else None
+        bio = info["bio"] if info["bio"] else None
+
+        if info["guild"]:
+            guild_name = (info["guild"]["name"]
```

Comment:
You have already checked that guild has some info. Why do you need to check that one more time with "name"?

Reply:
I thought that we might have some information about the "guild" and then we check the items inside the "guild"

Reply:
But maybe I'm wrong